chocolatey/plugins/modules/win_chocolatey_feature.ps1

```diff
@@ -39,26 +46,41 @@ $spec = Get-ModuleSpec
 $module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
 Set-ActiveModule $module
 
-$name = $module.Params.name
-$state = $module.Params.state
+$FeaturesToSet = if ($module.Params.features) {
+    $module.Params.features
+} else {
+    @{
+        $module.Params.name = $module.Params.state
+    }
+}
 
 $chocoCommand = Get-ChocolateyCommand
 $featureStates = Get-ChocolateyFeature -ChocoCommand $chocoCommand
 
-if ($name -notin $featureStates.Keys) {
-    $message = "Invalid feature name '$name' specified, valid features are: $($featureStates.Keys -join ', ')"
-    Assert-TaskFailed -Message $message
-}
+foreach ($Feature in $FeaturesToSet.GetEnumerator()) {
+    $name = $Feature.Key
+    $state = $Feature.Value
 
-$shouldBeEnabled = $state -eq "enabled"
-$isEnabled = $featureStates.$name
+    if ($name -notin $featureStates.Keys) {
+        $message = "Invalid feature name '$name' specified, valid features are: $($featureStates.Keys -join ', ')"
+        Assert-TaskFailed -Message $message
+    }
```

Comment:
initial offhand thought - would probably be better to collate a list of the features and values that were invalid so we can report _all_ of them rather than just the first one. Nobody likes playing whack-a-mole with error messages :D

Reply:
I like that! I was worried about compatibility for a lazy things like `-notin`, but will figure something / show that it works in PS2.0 environments!

Reply:
From memory, `-in` and `-notin` are in PSv2, it's `-contains` and `-notcontains` that are missing. 🙂 

Also, for this we're OK limiting support to PSv3 as that's Ansible's minimum requirement.

Reply:
First thought here is that collating the list is good here, as it's features and then are boolean. We're either turning something on or off, or you used a feature that's not in the list.

This is going to be trickier in config as you _can_ supply junk data and chocolatey will happily apply it. Unless that's been fixed in 1.x and I wasn't made aware. 

But doing something like `choco config set SomeJunkKey SomeJunkValue` will happily create that for ya.

Reply:
I feel like that's hard for us to validate in this scope - to an extent I feel that if Chocolatey doesn't, we shouldn't do so here.

Reply:
> can supply junk data

What do you mean by that @steviecoaster ?